gradle/project.gradle.kts

```diff
@@ -72,9 +72,7 @@ allprojects {
         options.isDeprecation = true
         options.compilerArgs = listOf(
                 "-Xlint:unchecked",
-                "--add-exports", "javafx.controls/com.sun.javafx.scene.control=ALL-UNNAMED",
-                "--add-exports", "javafx.controls/com.sun.javafx.scene.control.skin=ALL-UNNAMED",
-                "--add-exports", "javafx.web/com.sun.webkit=ALL-UNNAMED"
+                "--add-exports", "javafx.controls/com.sun.javafx.scene.control.skin=ALL-UNNAMED"
```

Comment:
Pls try recompiling all widgets as a 'test' that this works. I fear that WebReader will not compile.

Reply:
Please make sure the project is completely compiled (no cache, incremental compilation ect.).

Reply:
I have done that multiple times without problems already

Reply:
Ok thx for checking.